apps/src/aichat/redux/thunks/helpers/saveAiCustomization.ts

```diff
@@ -0,0 +1,95 @@
+// This is the "core" update logic that is shared when a student saves their
```

Comment:
Incomplete comment?
// model customizations (setup, retrieval, and "publish" tab)

Reply:
Oh good catch, this comment got split up when copy/pasting